.gitignore

```diff
@@ -1,5 +1,7 @@
 src/qinfer/version.py
 
+*.sublime-workspace
+
```

Comment:
My thought on that was that at least some files in ``.vscode`` aren't machine dependent, and can be used to specify things like how to build the docs. Since it's an open-source and cross-platform editor (that still feels weird to me), it may be worth providing some support? More generally, I have yet to form a coherent policy on project files for various editors and IDEs. Thoughts?